tools/packaging/kata-deploy/README.md

```diff
@@ -11,17 +11,28 @@ a node only if it uses either containerd or CRI-O CRI-shims.
 
 ### Install Kata on a running Kubernetes cluster
 
+#### Installing the latest image
+
+The latest image refers to pre-release and release candidate content.  For stable releases, please, use the "stable" instructions.
+
+```sh
+$ kubectl apply -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/kata-rbac/base/kata-rbac.yaml
+$ kubectl apply -f https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml
+```
+
+#### Installing the stable image
+
+The stable image refers to the last stable releases content.
```

Comment:
I think this would be a good place to tell people that if they use a tagged version of the repo then the stable image will match that version.   ex:  If you use the 2.1.1 tagged version of the `kata-deploy-stable.yaml` file then version 2.1.1 of the Kata runtime will be installed.